incubator/kitura/templates/default/Tests/ApplicationTests/RouteTests.swift

```diff
@@ -0,0 +1,154 @@
+import Foundation
+import Kitura
+import KituraNet
+import XCTest
+import AppsodyKitura
+
+@testable import Application
+
+class RouteTests: XCTestCase {
```

Comment:
Rename to something like 'BaseImageTests' and include OpenAPI endpoint test (metrics too?)